docs/cn/Running-Alluxio-Yarn-Integration.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: global
+title: Alluxio与YARN整合运行
+nickname: Alluxio与YARN整合
+group: Deploying Alluxio
+priority: 4
+---
+
+* 内容列表
+{:toc}
+
+本指南介绍在一个YARN集群中运行Alluxio作为一个应用的过程。自带的在EC2上运行Alluxio + YARN的教程，查阅[this guide](Running-Alluxio-on-EC2.html)
+
+注意：YARN不是非常适合像Alluxio这样的长时间运行的应用，我们推荐[these instructions](Running-Alluxio-Yarn-Standalone.html)来替代作为一个YARN应用来运行Alluxio。
+
+## 前期准备
+
+**一个运行的YARN的集群**
+
+**Alluxio在本地下载完成**
+
+```bash
+$ curl http://downloads.alluxio.org/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz | tar xz
+```
+
+## 构建与YARN的整合
+
+```bash
+$ mvn clean install -Dhadoop.version=<your hadoop version> -Pyarn -Dlicense.skip -DskipTests -Dfindbugs.skip -Dmaven.javadoc.skip -Dcheckstyle.skip
+```
+
+确保用你使用的Hadoop版本来替代<your hadoop version>
+
+## 配置
+通过一些特定的属性来定制Alluxio master和worker(比如，在各个worker节点上设置分层存储)，查阅[Configuration settings](Configuration-Settings.html)
+为了确保你的配置能够被ApplicationMaster和Alluxio master/workers都读到，将`alluxio-site.properties` 放到`/etc/alluxio/alluxio-site.properties`中。
+
+如果YARN不再`HADOOP_HOME`中，设置环境变量`YARN_HOME`为YARN的基础路径。
+
+## 运行Alluxio应用
+
+使用脚本`integration/yarn/bin/alluxio-yarn.sh`来启动Alluxio，这个脚本需要三个参数：
+
+1、需要启动的Alluxio workers的总数目（必须的）
```

Comment:
this has to be <pre>1.</pre> not <pre>1、</pre>, otherwise markdown will not recognize this. 
See https://guides.github.com/features/mastering-markdown/